compiler/src/dotty/tools/MainGenericRunner.scala

```diff
@@ -207,16 +212,18 @@ object MainGenericRunner {
       case ExecuteMode.Script =>
         val targetScript = Paths.get(settings.targetScript).toFile
         val targetJar = settings.targetScript.replaceAll("[.][^\\/]*$", "")+".jar"
-        val precompiledJar = Paths.get(targetJar).toFile
+        val precompiledJar = File(targetJar)
```

Comment:
Is this needed? Nio is newer and usually more respected that old io API, so I am curious

Reply:
> Is this needed? Nio is newer and usually more respected that old io API, so I am curious

This is not needed, but it now matches how File instanced str created everywhere else in `MainGenericRunner`, and it's also more efficient (by directly initializing a `File` rather than creating a `Path` and then converting it to a `File`).   However, perhaps we should leave it like it was.

Reply:
If you feel like `File` is OK, let it be then, I won't argue